web_invalid_tab/static/src/js/view_form.js

```diff
@@ -0,0 +1,62 @@
+openerp.web_invalid_tab = function(instance) {
+
+    var tab_selector = 'div[role="tabpanel"]';
+
+    function tab_link(tab) {
+        return $("a[href='#" + tab.attr('id') + "']");
+    }
+
+    function is_visible(tab, element) {
+        if ($(element).hasClass('oe_form_invisible') || element.style.display == 'none') {
```

Comment:
Try this case: Open a Order form, which has 3 tabs. In the third tab there is a required field. When you open the form the first tab is active, you fill the fields that you see and try to save. I have to highlight the third tab even the user never touch it (activate it), because inside has a field that should be filled.